packages/react-server-cli/src/parseCliArgs.js

```diff
@@ -13,7 +12,6 @@ export default (args = process.argv) => {
 			type: "number",
 		})
 		.option("host", {
-			default: "localhost",
```

Comment:
This is safe to just omit because it's defaulted somewhere deeper than `defaultOptions.js`?